noxfile.py

```diff
@@ -27,6 +27,7 @@ def test(session):
         "pytest",
         "--cov=installer",
         "--cov-fail-under=100",
+        "--cov-report=term-missing",
```

Comment:
😭 nox over tox

Reply:
That’s @pradyunsg, I just use whatever is available 🤷

Reply:
just nitpick, 😆 feel free to ignore; but felt as maintainer of tox must make the comment